### ISSUE TITLE
Evaluate usage of JSONassert instead of Approvals library

### ISSUE THREAD

**Issue by magicDGS** [ygojson/ygojson-tools#25]
Currently our main use case for testing with the approvals library is some json-outputs, that could be checked also with [JSONassert](http://jsonassert.skyscreamer.org/). As most of the tooling would be related to json format, looks like it might be worth to switch to a dedicated assert library for the format than a full approvals library that is for black-box testing.

In any case, we can keep the test dependency for approvals, as there might be other kind of acceptance tests for some non-json related tools (i.e., documentation, stats...)

**Comment by magicDGS** [comment]
Making it a higher priority, as changes in either the implementation of the approvals-library or the intellij toolbox make it a bit difficult to iterate over acceptance tests, as on my local system it triggers idea-diff, which hangs some process that keep piling up and open only after the IDE is gone.

Should report to the approvals-library or IntelliJ (although I believe that for the later the problem is alredy reported).

**Comment by magicDGS** [comment]
Also looks like quarkus dev is not able to re-run the tests (while implementing last part of #120), so we will need to remove the library if we wanna use these features.

**Comment by magicDGS** [comment]
As this might be more difficult and the only gain is to remove the dependency on approvaltest (which means finding a new way to do this kind of snapshot/acceptance tests), we will track the issue with quarkys in #134  in case that there is a faster workaround to proceed with the application logic development.

This isuse should be used to track and explore libraries to substitute the approvaltest one (with lower priority).
